### PR TITLE
Styling for both popup windows

### PR DESCRIPTION
**Pull request by RajeshwariJadhav**
- **style.css** is linked to both popup.html and preferred.html for uniformity.
- Both popup windows follow the color and font scheme of the admin console page.
- Navigation functionality added between Select OU page and popup home page.

## Popup Home Page
image

## Popup Select OU Page
image

### CODE REVIEW

**Comment by RajeshwariJadhav** [comment]
@googlebot I signed it!

**Comment by RajeshwariJadhav** [comment]
Added functionality to navigate between Select OU page and Popup home page.

File: STEP_Project/popup.html [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+	<link href="./style.css" rel="stylesheet" >
 </head>
 
 <body>
     <p>Preferred Entity Selector</p>
-    <button id = "addRemove">ADD/REMOVE PREFERRED ORG UNIT</button>
-    <button id = "select">SELECT A PREFERRED ORG UNIT</button>
+    <button id = "addRemove" class = "blueButton">ADD/REMOVE PREFERRED ORG UNIT</button>
+    <a href = preferred.html>
+    	<button id = "select" class = "blueButton">SELECT A PREFERRED ORG UNIT</button>
```
Comment (atishey-jain) [inline STEP_Project/popup.html:11]: Fix indentation.

File: STEP_Project/popup.html [hunk 2]
```diff
@@ -1,12 +1,15 @@
 <!DOCTYPE html>
 <html>
 <head>
+	<link href="./style.css" rel="stylesheet" >
```
Comment (atishey-jain) [inline STEP_Project/popup.html:4]: Fix indentation, seems like there's an extra tab here.

File: STEP_Project/popup.html [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
 </head>
 
 <body>
     <p>Preferred Entity Selector</p>
-    <button id = "addRemove">ADD/REMOVE PREFERRED ORG UNIT</button>
-    <button id = "select">SELECT A PREFERRED ORG UNIT</button>
+    <button id = "addRemove" class = "blueButton">ADD/REMOVE PREFERRED ORG UNIT</button>
+    <a href = preferred.html>
+    	<button id = "select" class = "blueButton">SELECT A PREFERRED ORG UNIT</button>
+	</a>
```
Comment (atishey-jain) [inline STEP_Project/popup.html:12]: Fix indentation.

**Comment by areeb1992** [comment]
Can you add a screenshot in the PR discription?

File: STEP_Project/style.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+p 
+{
+  font-family: 'Product Sans',Arial,sans-serif;
+  font-size: 16px;
+  text-align: center;
+}
+
+body 
+{
+  min-width:300px;
```
Comment (areeb1992) [inline STEP_Project/style.css:25]: [nit]: Add a single space after colon `:`. Check everywhere.

Comment (areeb1992) [inline STEP_Project/style.css:25]: [nit] Add a single space after colon `:`. Check everywhere.

File: STEP_Project/style.css [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  right:0;
+  text-decoration:none;
+  font-size: 20px;
+  color: rgba(69, 65, 65, 0.605); 
+}
+
+.radioButtons
+{
+  margin: 10px;
+}
```
Comment (areeb1992) [inline STEP_Project/style.css:43]: One empty line at the end of the file.

**Pull request merged**